pkg/subgraph/subgraph.yaml

```diff
@@ -134,7 +134,7 @@ templates:
 
   - kind: ethereum/contract
     name: CVStrategyV0_0
-    network: optimism
+    network: matic
```

Comment:
should be gnosis instead? also the chain id 100?

Reply:
I also matched the contract version for other chains so that was just the latest to be deployed but I dpeloyed to the all 4 prod chains (optimism, arbitrum, gnosis and polygon)